apps/taxes/src/pages/api/webhooks/checkout-calculate-taxes.ts

```diff
@@ -26,48 +40,34 @@ export default checkoutCalculateTaxesSyncWebhook.createHandler(async (req, res,
   const { payload } = ctx;
   logger.info({ payload }, "Handler called with payload");
 
-  const validation = calculateTaxesPayloadSchema.safeParse(payload);
-
-  if (!validation.success) {
-    logger.error({ error: validation.error.format() }, "Payload is invalid");
+  try {
+    verifyCalculateTaxesPayload(payload);
+    logger.info("Payload validated succesfully");
+  } catch (error) {
+    logger.error({ error: error }, "Payload is invalid");
     logger.info("Returning no data");
     return res.send({});
```

Comment:
Shouldnt you send actual error back? Its a bad request I assume

Reply:
I see `return res.send({});`. You should probably return 200 + a comment why + error message, `ok: false`?